modules/containers-signature-verify-enable.adoc

```diff
@@ -78,19 +79,90 @@ $ butane 51-worker-rh-registry-trust.bu -o 51-worker-rh-registry-trust.yaml
 $ oc apply -f 51-worker-rh-registry-trust.yaml
 ----
 
-. Create a Butane config and corresponding machine config which writes the configuration to disk on the control plane nodes:
+. Check that the worker machine config pool has rolled out with the new machine config:
```

Comment:
Change 'config' to 'configuration' for the cases in this section. (IBM Style GUide Word usage).
https://www.ibm.com/docs/en/ibm-style?topic=word-usage#i

Reply:
@tmalove Here, `machine config` and `machine config pool` are Kubernetes objects. We tend to use `config` when referring to these objects.